.gitlab-ci.yml

```diff
@@ -359,6 +359,15 @@ cargo-check-macos:
   tags:
     - osx
 
+test-prometheus-alerting-rules:
+  stage:                           test
+  image:                           paritytech/tools:latest
+  <<:                              *kubernetes-build
+  script:
+    - curl -L https://github.com/prometheus/prometheus/releases/download/v2.19.0/prometheus-2.19.0.linux-amd64.tar.gz --output prometheus.tar.gz
```

Comment:
Why not add it to the base image?

Reply:
A couple of reasons (not strong opinions though):

- I doubt any other user of the `paritytech/tools` image would use `promtool`.

- With the above in mind it adds an additional 49 MB to the image.

- When we want to update to a newer version of `promtool` we need to touch the image.

- The pipeline step is not the bottleneck of the stage (13s vs. 18 m and 46 s for *test-linux-stable*).

Reply:
We always download this file, for each run. We need to touch the yaml file for updating the version with this approach.

Reply:
I let @TriplEight decide.

Reply:
Now `promtool` is already in the `tools` image.